packages/styles/button.css

```diff
@@ -126,7 +126,7 @@ button.Link {
 .Button--tertiary {
   background-color: var(--button-background-color-secondary);
   color: var(--button-text-color-dark);
-  border: 1px solid var(--field-border-color);
+  border: 1px solid var(--gray-40);
```

Comment:
This should be using the scoped variable `--field-border-color`. If it needs to be different for light/dark modes the variable will need to be scoped to `:root` or `.cauldron--theme-dark` respectively.